src/redux/contacts/actions.ts

```diff
@@ -5,7 +5,7 @@ import { Contact } from '../../models/interfaces/contact'
 import axios from 'axios'
 
 export const getAllContacts = createAsyncThunk<any>('CONTACTS/GET_ALL_CONTACTS', async () => await axios.get('/api/contacts'));
-export const setNewContact = createAsyncThunk<any, Contact>("SET_NEW_CONTACT", async (data) => {
+export const setNewContact = createAsyncThunk<any, Contact>("CONTACTS/SET_NEW_CONTACT", async (data) => {

```

Comment:
This should go back to how it was and there should be a thunk for creating the contact. That way you can use `setNewContact` for the form and the creating thunk for the POST